intro-to-ruby.rb

```diff
@@ -1,14 +1,15 @@
 #create a string and store it in a variable
-
+name = "Hello, I'm Tiffany Fermin"
 #print out the value of the variable you just created
-
+puts name
 #print the sum of 185 and 14
+puts (185+14)
 
 #print the product of 1.8 and 3.6
-
+puts (1.8*3.6)
 #store the number 78 as a variable
-
+x = 78
 #print the average of the following numbers: 78, 55, 63, 98
-
+puts [(x+55+63+98)/4]
```

Comment:
don't use hard brackets here as they're used to denote arrays.

`puts (x+55+63+98)/4` works just fine